appcenter/AppCenter.d.ts

```diff
@@ -37,3 +37,9 @@ export function setEnabled(enabled: boolean): Promise<void>;
 export function setUserId(userId: string | null): Promise<void>;
 export function getInstallId(): Promise<string>;
 export function setCustomProperties(properties: CustomProperties);
+export function setAuthToken(authToken: string | null): void;
+export function setAuthTokenListener(listener: AuthTokenListener) : void;
+
+export interface AuthTokenListener {
+    acquireAuthToken: () => string | null;
```

Comment:
```suggestion
    acquireAuthToken?: () => Promise<string | null> | string | null;
```

Not sure about syntax but we are awaiting this so either a promise or the direct value works.

Reply:
Awaiting on primitive types work, so technically we actually support both promise and directly the value.

The comment is thus not addressed (also the optional aspect of it is still true).